helper/chart_data.js

```diff
@@ -8,23 +8,33 @@ const default_no_of_assessment_submissions_threshold = 3;
 const default_entity_score_api_threshold = 5;
 
 //function for instance observation final response creation
-exports.instanceReportChart = async function (data,reportType) {
-    var obj;
-    var multiSelectArray = [];
-    var matrixArray = [];
-    var order = "questionExternalId";
+exports.instanceReportChart = async function (data,reportType = "") {
+    let obj;
+    let multiSelectArray = [];
+    let matrixArray = [];
+    let order = "questionExternalId";
     let actualData = data;
+    let type;
 
     try {
-        // obj is the response object which we are sending as a API response   
-        obj = {
-            entityName: data[0].event.schoolName,
-            observationName: data[0].event.observationName,
-            observationId: data[0].event.observationId,
-            entityType: data[0].event.entityType,
-            entityId: data[0].event.school,
-            districtName: data[0].event.districtName,
-            response: []
+        if (reportType && reportType == filesHelper.survey) {
+            type = filesHelper.survey;
+            obj = {
+                solutionName: data[0].event.solutionName,
+                response: []
+            }
+        }
+        else {
+            type = filesHelper.observation;
+            obj = {
+                entityName: data[0].event.schoolName,
```

Comment:
@deepakr123 - Please confirm is this is being used. If not let's not send it.

Reply:
resolved